src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+
+  if (days < 3) {
+    cost *= days;
+  }
+
+  if (days < 7 && days >= 3) {
```

Comment:
Ты уже вышла из цикла по условию days < 3;
```suggestion
  if (days < 7) {
```